### ISSUE TITLE
Enable bypassing NamedParameter

### ISSUE THREAD

**Issue by jasonb5** [ESGF/esgf-compute-api#4]
Provide a more implicit way to pass named parameters. Allow users to define named parameters as dict's when passing to an operation or process. 

``` python
op = Operation('averager', 'ophidia', [v0, NamedParameter('axes', 't')], var_name='avg_oph')
op = Operation('averager', 'ophidia', [v0, NamedParameter('axes', 't'), NamedParameter('bins', 't', 'mon', 'year')], var_name='avg_oph')
```

would become

``` python
op = Operation('averager', 'ophidia', [v0, { 'axes': 't' }], var_name='avg_oph')
op = Operation('averager', 'ophidia', [v0, { 'axes': 't', 'bins': ['t', 'mon', 'year']}], var_name='avg_oph')
```

**Comment by jasonb5** [comment]
Fixed by #8 

Need to add tests

**Comment by jasonb5** [comment]
Added tests and has been merged to master
